tests/backup_helper.go

```diff
@@ -145,6 +145,11 @@ var (
 	GlobalAWSLockedBucketName   string
 	GlobalAzureLockedBucketName string
 	GlobalGCPLockedBucketName   string
+	GlobalPreRuleName           string
+	GlobalPostRuleName          string
+	GlobalPreRuleUid            string
+	GlobalPostRuleUid           string
+	FlagUseGlobalRule           bool
```

Comment:
nit: Can we rename this to `UseGlobalRule` or `GlobalRuleFlag` ?

Reply:
Done